wheatley/row_generation/helpers.py

```diff
@@ -5,9 +5,10 @@
 import itertools
 import re
 
+from wheatley.types import Places, Row
 from wheatley.bell import Bell, BELL_NAMES
 
-_CROSS_PN = []
+_CROSS_PN: List[bool] = []
```

Comment:
Should this be 
```suggestion
_CROSS_PN: Places = []
```
i.e. conceptually a `List[int]`

Reply:
Which might help the type error commented below?

Reply:
Nice spot - I still had about 20 error messages to fix on my local branch when I turned in for the night, which I'll fix today...